tests/test_codebase/test_mmpose/test_mmpose_models.py

```diff
@@ -0,0 +1,230 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import mmcv
+import numpy as np
+import pytest
+import torch
+
+from mmdeploy.codebase import import_codebase
+from mmdeploy.utils import Backend, Codebase
+from mmdeploy.utils.test import WrapModel, check_backend, get_rewrite_outputs
+
+import_codebase(Codebase.MMPOSE)
+
+input = torch.rand(1)
+
+
+def get_top_down_heatmap_msmu_head_model():
+    from mmpose.models.heads import TopdownHeatmapMSMUHead
+    model = TopdownHeatmapMSMUHead((32, 48), unit_channels=2,
+                                   loss_keypoint=dict(type='JointsMSELoss',
+                                                      use_target_weight=False))
+
+    model.requires_grad_(False)
+    return model
+
+
+@pytest.mark.parametrize(
+    'backend_type', [Backend.DEFAULT])
```

Comment:
default backend will not run inference, better explicitly set backend as onnxruntime and TensorRT 
https://github.com/open-mmlab/mmdeploy/blob/003cf1bfd7cec20f9c296825dab0c2681c3fc5e6/mmdeploy/utils/test.py#L474